component/component/main.jsonnet

```
@@ -39,69 +111,5 @@ local secrets = [
 
   secrets: std.filter(function(it) it != null, secrets),
 
-  cronjob: {
-    kind: 'CronJob',
-    apiVersion: 'batch/v1',
-    metadata: {
-      name: alias,
-      namespace: paramsACR.namespace,
-      labels+: labels,
-    },
-    spec: {
-      concurrencyPolicy: 'Forbid',
-      failedJobsHistoryLimit: 5,
-      jobTemplate: {
-        spec: {
-          template: {
-            spec: {
-              restartPolicy: 'OnFailure',
-              containers: [
-                {
-                  name: 'exoscale-metrics-collector-backfill',
-                  image: collectorImage,
-                  args: [
-                    'objectstorage',
-                  ],
-                  envFrom: [
-                    {
-                      secretRef: {
-                        name: credentials_secret_name
-                      }
-                    }
-                  ],
-                  env: [
-                    {
-                      name: 'password',
-                      valueFrom: {
-                        secretKeyRef: {
-                          key: 'password',
-                          name: 'reporting-db',
-                        },
-                      },
-                    },
-                    {
-                      name: 'username',
-                      valueFrom: {
-                        secretKeyRef: {
-                          key: 'username',
-                          name: 'reporting-db',
-                        },
-                      },
-                    },
-                    {
-                      name: 'ACR_DB_URL',
-                      value: 'postgres://$(username):$(password)@%(host)s:%(port)s/%(name)s?%(parameters)s' % paramsACR.database,
-                    },
-                  ],
-                  resources: {},
-                },
-              ],
-            },
-          },
-        },
-      },
-      schedule: params.schedule,
-      successfulJobsHistoryLimit: 3,
-    },
-  },
+  cronjobs: [c for c in cronjobs if c != null],
```

Instead of the array approach you could also do this:
```suggestion
  objectstorageConjob: cronjob(...),
  [if params.dbaas.enabled then "dbaasCronjob"]: cronjob(...),
```

Might be a bit cleaner